datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx

```diff
@@ -40,6 +54,19 @@ export const DatasetStatsSummary = ({
             <StatText>
                 <TableOutlined style={{ marginRight: 8, color: ANTD_GRAY[7] }} />
                 <b>{formatNumberWithoutAbbreviation(rowCount)}</b> rows
+                {(!!columnCount && (
+                    <>
+                        , `<b>{formatNumberWithoutAbbreviation(rowCount)}</b> columns
```

Comment:
should be `columnCount` here instead of `rowCount`.

Also nit: I've noticed this in a good amount of places in the app, and while this situation is uncommon, if `columnCount` is 1, we should say singular "column" instead. Elsewhere I've simply replaced your "s" above with `{columnCount > 1 && 's'}`

Reply:
yes good idea!